conf/vertebrates/mlss_conf.xml

```diff
@@ -34,98 +34,275 @@
 
-    <one_vs_all method="LASTZ_NET" ref_genome="ovis_aries" against="Caprinae" ref_amongst="Cetartiodactyla"/>
+    <one_vs_all method="LASTZ_NET" ref_genome="sus_scrofa" against="Sus"/>
+    <one_vs_all method="LASTZ_NET" ref_genome="monodelphis_domestica" against="Metatheria"/>
+    <one_vs_all method="LASTZ_NET" ref_genome="ornithorhynchus_anatinus" against="Prototheria"/>
 
-    <!-- Early vertebrates and other chordates -->
-    <one_vs_all method="LASTZ_NET" ref_genome="ciona_intestinalis" against="Ciona" ref_amongst="Cyclostomata"/>
-    <one_vs_all method="LASTZ_NET" ref_genome="petromyzon_marinus" against="Cyclostomata" ref_amongst="Ciona"/>
+    <!-- Sauropsids -->
+    <one_vs_all method="LASTZ_NET" ref_genome="gallus_gallus">
+      <species_set>
+        <taxonomic_group taxon_name="Sauropsida"/>
+        <!-- Remove all the taxa that have a better reference than chicken -->
+        <taxonomic_group taxon_name="Iguania" exclude="1"/>
+      </species_set>
+    </one_vs_all>
+
+    <one_vs_all method="LASTZ_NET" ref_genome="anolis_carolinensis" against="Iguania"/>
 
-    <!-- These genomes are currently the only ones in their clades, so no alignment within the "against"/"species_set" -->
-    <!-- However they will be aligned against the other reference genomes in "ref_amongst" -->
-    <one_vs_all method="LASTZ_NET" ref_genome="equus_caballus" against="Perissodactyla" ref_amongst="Laurasiatheria"/>
-    <one_vs_all method="LASTZ_NET" ref_genome="sus_scrofa" against="Suina" ref_amongst="Laurasiatheria"/>
-    <one_vs_all method="LASTZ_NET" ref_genome="anolis_carolinensis" against="Lepidosauria" ref_amongst="Sauropsida"/>
-    <one_vs_all method="LASTZ_NET" ref_genome="pelodiscus_sinensis" against="Testudines" ref_amongst="Sauropsida"/>
-    <one_vs_all method="LASTZ_NET" ref_genome="xenopus_tropicalis" against="Amphibia" ref_amongst="Euteleostomi"/>
-    <one_vs_all method="LASTZ_NET" ref_genome="ornithorhynchus_anatinus" against="Prototheria" ref_amongst="Euteleostomi"/>
-    <one_vs_all method="LASTZ_NET" ref_genome="latimeria_chalumnae" ref_amongst="Euteleostomi">
+    <!-- Fishes -->
+    <one_vs_all method="LASTZ_NET" ref_genome="oryzias_latipes">
       <species_set>
-        <!--Coelacanths and lungfishes -->
-        <taxonomic_group taxon_name="Sarcopterygii"/>
+        <!-- Include urochordates, cephalochordates, cyclostomes, cartilaginous fishes, ray-finned fishes, coelacanths and lungfishes -->
+        <taxonomic_group taxon_name="Chordata"/>
         <taxonomic_group taxon_name="Tetrapoda" exclude="1"/>
       </species_set>
     </one_vs_all>
+
+    <!-- Early vertebrates and other chordates -->
+    <one_vs_all method="LASTZ_NET" ref_genome="xenopus_tropicalis" against="Amphibia"/>
+    <one_vs_all method="LASTZ_NET" ref_genome="ciona_intestinalis" against="Ciona" ref_amongst="Cyclostomata"/>
+    <one_vs_all method="LASTZ_NET" ref_genome="petromyzon_marinus" against="Cyclostomata" ref_amongst="Ciona"/>
```

Comment:
> The best lamprey alignment is against hagfish (then zebrafish) but the best hagfish alignment is against zebrafish. Elephant shark alignment has a slightly better alignment with human, but zebrafish is OK. Reedfish (which is an EPO) has a better alignment with zebrafish than human.
> The best coelacanth alignments are against chicken and platypus. All other alignments are equivalent. It's kinda the same for xenopus.
> 
> Overall, if we need to choose one, I would say:
> 
> * chicken against all Sarcopterygii except Amniota (Amniota = Sauropsida + Mammalia, which are both already addressed). That will capture coelacanth and xenopus
> * zebrafish against all Chordata, except Euteleostomi (Euteleostomi = Sarcopterygii (see the previous rule) + Actinopterygii (medaka)) and Cyclostoma and Ciona (they both have alignments against each other). This complex rule only matches the elephant shark at the moment, but it's there as a catch-all in case we get other basal vertebrates / chordates, given the fact that zebrafish has OK alignments with the species we have


> @dthybert . I don't understand what you mean re. zebrafish and medaka: which one do you want to use in EPO-Extended ?

For EPO extended in fish the rule you wrote above is fine (I missed it yesterday when I was reviewing). Zebra fish is such a big reference for the fish community that anyway if its genome is updated it will have to be treated like the mouse and the human.